backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.adoc

```diff
@@ -8,23 +8,33 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-You install the Openshift API for Data Protection (OADP) with Multicloud Object Gateway (MCG) by installing the OADP Operator, retrieving the MCG credentials, and then installing the Data Protection Application.
+You install the Openshift API for Data Protection (OADP) with Multicloud Object Gateway (MCG) by installing the OADP Operator, creating a `Secret` object, and then installing the Data Protection Application.
 
-MCG is a component of OpenShift Container Storage (OCS). You can configure MCG as a backup location.
+MCG is a component of OpenShift Container Storage (OCS). You configure MCG as a backup location.
 
-To install the OADP Operator in a restricted network environment, you must first disable the default OperatorHub sources and mirror the Operator catalog. See xref:../../../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks] for details.
+:FeatureName: The `CloudStorage` API for S3 storage
+include::modules/technology-preview.adoc[]
+
+You can configure a snapshot location if you have a cloud provider that supports either a native snapshot API or Container Storage Interface (CSI) snapshots.
```

Comment:
For Container Storage Interface (CSI) snapshots, it doesn't need to configure a snapshot location, it just need to add csi plugin and enable CSI
```
spec:
  configuration:
    velero:
      defaultPlugins:
      - openshift
      - aws
      - csi     ## add CSI plugin
    featureFlags:
    - EnableCSI    ## eanble CSI
```